tools/cellxgene_census_builder/tests/test_workflow_steps.py

```diff
@@ -8,6 +8,7 @@
 from cellxgene_census_builder.release_manifest import CensusVersionDescription
 
 TEST_BUCKET_PATH = "s3://bucket/path"
+TEST_BUCKET_PREFIX = "/path"
```

Comment:
as these two constants must be in sync, might be better to do this:
```
TEST_PATH = "/path"
TEST_BUCKET_URI = f"s3://bucket/{TEST_PATH}"
```

Note that I also renamed them to match their actual contents.

Reply:
I see same pattern in other test files - might want to fix all of them

Also, this is a _nit_